src/menu.css.ts

```diff
@@ -1,40 +1,41 @@
-import {style, createVar} from '@vanilla-extract/css';
+import {createVar, style} from '@vanilla-extract/css';
 import {sprinkles} from './sprinkles.css';
 import {vars as colorsVars} from './skins/skin-contract.css';
 
 const top = createVar();
 const bottom = createVar();
-const right = createVar();
+const left = createVar();
 const width = createVar();
-const transformOrigin = createVar();
 const maxHeight = createVar();
+const transformOrigin = createVar();
 
 export const vars = {
     top,
     bottom,
-    right,
+    left,
     width,
-    transformOrigin,
     maxHeight,
+    transformOrigin,
 };
 
 export const menuContainer = style([
     sprinkles({
         padding: 0,
-        position: 'absolute',
         background: colorsVars.colors.backgroundContainer,
         borderRadius: 8,
     }),
     {
         top,
         bottom,
-        right,
+        left,
         width,
-        transformOrigin,
         maxHeight,
+        transformOrigin,
+        display: 'flex',
+        flexDirection: 'column',
+        position: 'absolute',
```

Comment:
move to sprinkles (position was already there before), is display flex needed?

Reply:
Display flex is needed to make the menu items adopt all the horizontal space available for the menu.

Reply:
Moving these props to sprinkles ;)